sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -454,6 +463,34 @@ public void testReadingWithKeyRange() throws Exception {
     assertThat(suffixRows, hasItems(middleRows.toArray(new Row[]{})));
   }
 
+  /**
+   * Tests reading three key ranges with one read.
+   */
+  @Test
+  public void testReadingWithKeyRanges() throws Exception {
+    final String table = "TEST-KEY-RANGE-TABLE";
+    final int numRows = 1001;
+    List<Row> testRows = makeTableData(table, numRows);
+    ByteKey startKey1 = ByteKey.copyFrom("key000000100".getBytes());
```

Comment:
In this test the long prefix is probably not contributing to the test coverage. How about keys simply like "1", "2", etc.?

Reply:
Agree, though the prefix is generated from makeTableData(), which is commonly used throughout the test. I can make keys like "key000000001", but to get keys like "1" i'd have to recreate functionality already in makeTableData(), no?